migration/migration.c

```diff
@@ -1403,7 +1417,8 @@ MigrationState *migrate_init(const MigrationParams *params)
 
     s->total_time = qemu_clock_get_ms(QEMU_CLOCK_REALTIME);
     s2->total_time = qemu_clock_get_ms(QEMU_CLOCK_REALTIME);
-
+    s->ft_state = CUJU_FT_OFF;
```

Comment:
?

Reply:
if no initialize s->ft_state to CUJU_FT_OFF, after cuju-migrate-cancel the state would become CUJU_FT_TRANSACTION_FLUSH_OUTPUT and occur error:
qemu-system-x86_64: migration/migration.c:133: migrate_assert_ft_state_change: 
Assertion `t == CUJU_FT_TRANSACTION_FLUSH_OUTPUT' failed.